docker/edge/http/edge_stop.sh

```diff
@@ -0,0 +1,2 @@
+docker stop edgebroker && docker rm $_
+docker stop edgeworker && docker rm $_
```

Comment:
do newline is not needed at the end like in above script edge_start.sh is having.

Reply:
Comment Incorporated